### PR TITLE
Add Union Bank API to Module

### PR DESCRIPTION
**Pull request by mezlet**
#### What does this PR do?
- Add union bank endpoints to module
- Add test
- Add documentation
- RenameNIBSS and Sterling module folder

### CODE REVIEW

File: src/Union/README.md [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+$this->instance1->GenerateToken(
+    '', 
+   'your sandbox_key', 
+    [
+       	'client_secret' => 'secret',
+        'client_id' => 'web01',
+        'grant_type' =>	'password',
+        'username' => 'ubnclient01',
+        'password' => 'w$777'
+    ]);
```
Comment (akinjide) [inline src/Union/README.md]: Format properly

File: src/Union/README.md [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+
+$instance1->Customer(
+    '', 
+    'your sandbox key', 
+    [
+       'accountNumber' =>'+0000791200',
+       'accountType' =>'CASA',
+    ],
+    'access_token'
+    );
```
Comment (akinjide) [inline src/Union/README.md]: Format properly here too and everywhere else necessary

File: tests/Mock/Union.php [hunk 3]
```diff
@@ -0,0 +1,238 @@
...
+    {
+        return [
+            'host' => 'hbvhjbh',
+            'sandbox_key' => $this->sandbox_key,
+            'payload' => [
+                'client_secret' => 'secret',
+                'client_id ' => 'web01',
+                'grant_type' => 'password',
+                'username' => 'ubnclient01',
+                'password' => 'w$777'
```
Comment (akinjide) [inline tests/Mock/Union.php]: use random data

File: tests/Mock/Union.php [hunk 4]
```diff
@@ -0,0 +1,238 @@
...
+                'customerNumber' =>    '004607380',
+                'accountClass' => 'CA_044',
+                'accountCurrency' => 'NGN',
+                'accountType' => 'Current',
+                'availableBalance' => '5861',
+                'customerAddress' => '29 ALIU STREET OFF IJOKA ROAD AKURE LAGOS',
+                'customerEmail' => 'myawesoft@yahoo.com',
+                'customerPhoneNumber' => '08032227456'
+            ]
+        ];
```
Comment (akinjide) [inline tests/Mock/Union.php]: use random data

File: tests/Mock/Union.php [hunk 5]
```diff
@@ -0,0 +1,238 @@
...
+                    'accountType' => 'current',
+                    'availableBalance' => '5861',
+                    'customerAddress' => '29 ALIU STREET OFF IJOKA ROAD AKURE LAGOS',
+                    'customerEmail' => 'myawesoft@yahoo.com',
+                    'customerPhoneNumber' => '08032227456',
+                ],
+                'customer' => [
+                    'code' => '00',
+                    'message' => 'Customer Enquiry Successful',
+                    'country' => 'NG',
```
Comment (akinjide) [inline tests/Mock/Union.php]: use random data here too

**Review by akinjide — CHANGES_REQUESTED** [review]

File: tests/Mock/Union.php (hunk 4 above)
Comment (mezlet) [reply tests/Mock/Union.php]: done

File: tests/Mock/Union.php (hunk 5 above)
Comment (mezlet) [reply tests/Mock/Union.php]: done

File: tests/Mock/Union.php (hunk 3 above)
Comment (mezlet) [reply tests/Mock/Union.php]: done

File: src/Union/README.md (hunk 2 above)
Comment (mezlet) [reply src/Union/README.md]: done

File: src/Union/README.md (hunk 1 above)
Comment (mezlet) [reply src/Union/README.md]: done

**Review by akinjide — APPROVED** [review]

**Pull request merged**